slf4j-simple/src/test/java/org/slf4j/impl/SimpleLoggerTest.java

```diff
@@ -40,6 +40,7 @@
     private static final String MESSAGE_FORMAT = "This is a format: {}";
     private static final String PARAMETER = "This is an argument";
     private static final String LOGGER_NAME = "some.logger";
+    private static final String EXPECTED_FORMATTED_MESSAGE = "This is a format: " + PARAMETER;
```

Comment:
I would suggest to move this right after `MESSAGE_FORMAT`, so it is more obvious that these two are related.